src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -298,7 +317,117 @@
+                    var tabs = tabGroups[i].querySelectorAll('.tab');
+                    var tabNavigation = document.createElement('ul');
+                    tabNavigation.setAttribute('class', 'tab-navigation');
+
+                    for (var j = 0; j < tabs.length; j++) {
+                        var tabId = 'tab-' + i + '-' + j;
+                        var tabTitle = tabs[j].querySelector('.tab-title').innerHTML;
+
+                        var tabNavigationItem = document.createElement('li');
+                        tabNavigationItem.setAttribute('data-tab-id', tabId);
+                        tabNavigationItem.setAttribute('class', j == 0 ? 'active' : '');
+                        tabNavigationItem.innerHTML = tabTitle;
+                        tabNavigation.appendChild(tabNavigationItem);
+
+                        var tabContent = tabs[j].querySelector('.tab-content');
+                        tabContent.parentElement.setAttribute('id', tabId);
+                    }
+
+                    tabGroups[i].insertBefore(tabNavigation, tabGroups[i].firstChild);
+                }
+
+                /* display the active tab and add the 'click' event listeners */
+                for (i = 0; i < tabGroups.length; i++) {
+                    tabNavigation = tabGroups[i].querySelectorAll('.tab-navigation li');
+
+                    for (j = 0; j < tabNavigation.length; j++) {
+                        tabId = tabNavigation[j].getAttribute('data-tab-id');
+                        document.getElementById(tabId).querySelector('.tab-title').style.display = 'none';
+
+                        if (tabNavigation[j].classList.contains('active')) {
+                            document.getElementById(tabId).style.display = 'block';
+                        } else {
+                            document.getElementById(tabId).style.display = 'none';
+                        }
+
+                        tabNavigation[j].addEventListener('click', function(e) {
+                            var activeTab = e.target || e.srcElement;
+
+                            /* needed because when the tab contains HTML contents, user can click */
+                            /* on any of those elements instead of their parent '<li>' element */
+                            while (activeTab.tagName.toLowerCase() != 'li') {
+                                activeTab = activeTab.parentNode;
+                            }
+
+                            for (var k = 0; k < tabNavigation.length; k++) {
+                                var tabId = tabNavigation[k].getAttribute('data-tab-id');
+                                document.getElementById(tabId).style.display = 'none';
+                                tabNavigation[k].classList.remove('active');
+                            }
+
+                            activeTab.classList.add('active');
+                            var activeTabId = activeTab.getAttribute('data-tab-id');
```

Comment:
I'm afraid I don't understand this comment. Can you please show me the new code I should use instead of `var activeTabId = activeTab.getAttribute('data-tab-id');`? Thanks!